lib/trino-filesystem-gcs/src/test/java/io/trino/filesystem/gcs/TestDefaultGcsFileSystem.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.filesystem.gcs;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
+
+@EnabledIfEnvironmentVariable(named = "GOOGLE_APPLICATION_CREDENTIALS", matches = ".+")
```

Comment:
Move to ci.yml, see https://github.com/trinodb/trino/commit/cabc7c7ae89 for example.
We have a contract for downstream projects and fork owners (idnvidual contributors) that whenever you define a `CI_SKIP_SECRETS_PRESENCE_CHECKS` secret, all secrets-requiring tests are attempted. 
cc @losipiuk

Reply:
Done, lmk if that's ok - reusing the gcp secret.